hardware/nvidia/t23x/nv-public/6.0/0001-overlay-enable-d4xx-camera-for-Orin-jp6.patch

```diff
@@ -5,18 +5,19 @@ Subject: [PATCH] overlay: enable d4xx camera for Orin jp6
 
 Signed-off-by: Dmitry Perchanov <dmitry.perchanov@intel.com>
 ---
- overlay/Makefile | 1 +
- 1 file changed, 1 insertion(+)
+ overlay/Makefile | 2 ++
+ 1 file changed, 2 insertion(+)
 
 diff --git a/overlay/Makefile b/overlay/Makefile
 index c88bbe2..3f6354e 100644
 --- a/overlay/Makefile
 +++ b/overlay/Makefile
-@@ -60,6 +60,7 @@ dtbo-y += tegra234-p3767-camera-p3768-imx219-A.dtbo
+@@ -60,6 +60,8 @@ dtbo-y += tegra234-p3767-camera-p3768-imx219-A.dtbo
  dtbo-y += tegra234-p3767-camera-p3768-imx219-imx477.dtbo
  dtbo-y += tegra234-p3767-camera-p3768-imx477-C.dtbo
  dtbo-y += tegra234-p3767-camera-p3768-imx477-A.dtbo
 +dtbo-y += tegra234-camera-d4xx-overlay.dtbo
++dtbo-y += tegra234-camera-d4xx-overlay-dual.dtbo
```

Comment:
Jenkins build fails
Maybe this need a fix?
**https://github.com/IntelRealSense/librealsense-deploy/blob/019ded52f5ca54175985b293a3440aaeadbb442e/jenkins/LRS_Jetson_mipi_usb_driver_jp6.groovy#L247**


Build: https://rsjenkins.iil.intel.com/job/D4xx_Kernel_Module_Jetson_with_CV_scan_option_JP6/205/